### PR TITLE
Added a c-format target to the Makefile

### PR DESCRIPTION
**Pull request by drpdr**

### CODE REVIEW

**Comment by drpdr** [comment]
bors try

**Comment by drpdr** [comment]
Ready for review.

File: Makefile [hunk 1]
```diff
@@ -23,5 +24,8 @@ run-%: bin/% $$(wildcard %.c) $$(wildcard %.cpp)
 	scp -P $(SSHPORT) $(word 2,$^) bin/$(<F) root@127.0.0.1:/root
 	ssh -p $(SSHPORT) root@127.0.0.1 -t '/root/$(<F)'
 
+c-format:
```
Comment (ltratt) [inline Makefile:27]: I think we should either call this `cformat` (I always forget `-` characters) or `clang-format` (on the basis that that's the actual tool being run). I have no preference for which we go for.

Comment (drpdr) [reply Makefile:27]: Agreed.

Comment (drpdr) [reply Makefile]: bors try

Comment (drpdr) [reply Makefile]: FIXED in d1bc2bb.

Comment (drpdr) [reply Makefile]: But I believe I have made a mistake. Sorry.

**Comment by drpdr** [comment]
Ready for review.

**Pull request merged**